src/main/resources/db/migration/V1__baseline.sql

```diff
@@ -0,0 +1,86 @@
+CREATE TABLE supplier
```

Comment:
could we sort these creations a little more, either by type or by alphabet (mainly thinking the `acquirer_subscription_enrichment_field` should be after the `acquirer_subscription`)